src/gui/editors/AutomationEditor.cpp

```diff
@@ -1263,7 +1266,7 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 				int note_width = lenTicks * m_ppb / TimePos::ticksPerBar();
 				int keyRange = maxKey - minKey;
 
-				if (keyRange < MIN_NOTE_RANGE) 
+				if (keyRange < MIN_NOTE_RANGE)
```

Comment:
No need to change this line.

Reply:
My current editor does auto formatting so it can change something that I don't know. I'll try turning it off

Reply:
I know VSCode has a feature that automatically trims out whitespace (assuming you are using that editor). While this is good in principle, you get unrelated changes within `git blame`, which could be confusing.